src/components/RecipeEdit.vue

```diff
@@ -16,6 +16,15 @@
                 :field-label="t('cookbook', 'Description')"
                 :suggestion-options="allRecipeOptions"
             />
+            <EditInputField
+                v-model="recipe['citation']"
```

Comment:
I am not so sure about this usage of the `citation` field. According to the [schema.org standard](https://schema.org/citation), This field seems inappropriate according to the definition of this field:

> A citation or reference to another creative work, such as another publication, web page, scholarly article, etc.

This does not match the use case, especially with the _Grandma Betty_ placeholder content. How about using `author` or the `creator` fied.

I find it hard to quickly merge this PR now and soon have to create some sort of migration/fix that changes the code a posteriori.

Reply:
The "not so sure" part is true for the other fields to. So thank you for initiating the discussion! 

According to schema.org the `author` and `creator` fields are the same 

> `creator`: The creator/author of this CreativeWork. This is the same as the Author property for CreativeWork.

However, I tend towards the creator being the one that creates the recipe in the cookbook, i.e., the Nextcloud user (analog to the fact that the `dateCreated` field is the non-user-configurable date the recipe was created, and not the date when grandma invented the recipe). This could become relevant when real sharing of cookbooks, recipes, collections, ... between users becomes a thing. It would allow filtering recipes by the user who entered the recipe in the cookbook app. As usual, we could store this info outside of the JSON ...

On the other hand, what I, as a creator, am doing when creating the recipe is citing "My grandma", who told me about the  list of ingredients and steps to take, but not necessarily the exact wording of each instruction, or the exact format for entering ingredients. I also might add some more comments on how grandma used to watch birds in the front yard while stirring the sauce but still cite her because she originally had the idea.

I do see the point though, that when I want to cite 

```
The Science of Cooking - Every Question Answered to Perfect Your Cooking,
 Stuart Farrimond, Dorling Kindersley Limited, 2017
 ``` 

the approach with "by XXX" falls a bit short and the `creator` (or `author`) field would make it much clearer what to expect from it, i.e., the "by XXX" syntax could be expected. 

One option would be to get rid of the word "by" and leave it up to the user to format the string to be shown using Markdown, which would also immediately allow linking. However my idea of moving the `url` property to a href of the citation to clean up the UI when viewing a recipe would then be off the table. I think, it's the usual consideration configurability vs. cleanness/ease of use.

Wrapping up, I see how both options have advantages and disadvantages. I'm looking forward for your opinion ;)

Reply:
The general problem I see is the distinction in schema.org between a natural person/organization and a third (creative) work. The former is referencable by `author`/`creator`, while the latter is referenced by `citation`. In my understanding, these associations are mutually exclusive in some sort.

How about the following idea: We could have both fields (`citation` and `author`). One could be used as a source (in the sense of a reference), that would be a `citation`, while Grandma's best recipes could be declared using `author`. Would that work?

Of course, the UI must be thought through here as well. We would have 4 cases (no field, either field is set, or both fields are set). If none is set, we are at status quo. With one field set, your approach might work well (_by author_ or _from citation_ for example). With both present, we could think about it but I would then go with _by author_ and add a reference later on like the URL source currently.